src/user.rs

```diff
@@ -242,6 +250,7 @@ impl User<Registered> {
                 })
             }
             Err(e) => {
+                println!("LOGIN ERROR {} {}", e, self.id.localpart());
```

Comment:
I think we should avoid using `println`s as it can break the progress bars.

use `log::error` or `log::info` instead

Reply:
Fixed it